sale_promotion_rule/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Benoît GUILLOT <benoit.guillot@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Promotion Rule",
+    "summary": "Module to manage promotion rule on sale order",
+    "version": "10.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Sale",
+    "website": "https://akretion.com",
+    'author': 'Anybox, ACSONE SA/NV,Odoo Community Association (OCA)',
```

Comment:
this is strange: we have akretion in the website but not in the author

Reply:
I'll fix it... Should be Akretion in place of Anybox...

Reply:
@rvalyi fixed in https://github.com/OCA/sale-workflow/pull/729/commits/6fb82acd4c3538132d00a843acb58fe92afbb803